MediaBrowser.Providers/Plugins/Tmdb/TmdbUtils.cs

```diff
@@ -148,6 +148,12 @@ public static string NormalizeLanguage(string language)
 
             if (parts.Length == 2)
             {
+                // TMDB doesn't support Switzerland (de-CH, it-CH or fr-CH) so use the language (de, it or fr) without country code
+                if (string.Equals(parts[1], "ch", StringComparison.Ordinal))
```

Comment:
Are we sure this is always lowercase?
And maybe add a test for this

Reply:
Sorry I was confused. In theory it should always be upper case but to be safe I changed it to ignore case